librz/core/cmd/cmd_api.c

```diff
@@ -1110,15 +1115,20 @@ static void do_print_child_help(RzCmd *cmd, RzStrBuf *sb, const RzCmdDesc *cd, c
 	rz_strbuf_appendf(sb, "%s\n", pal_reset);
 }
 
-static void print_child_help(RzCmd *cmd, RzStrBuf *sb, RzCmdDesc *cd, size_t max_len, bool use_color) {
-	do_print_child_help(cmd, sb, cd, cd->name, cd->help->summary ? cd->help->summary : "", true, max_len, use_color);
+static void print_child_help(RzCmd *cmd, RzStrBuf *sb, RzCmdDesc *cd, size_t max_len, const char *vertical_line, bool use_color) {
+	do_print_child_help(cmd, sb, cd, cd->name, cd->help->summary ? cd->help->summary : "", vertical_line, true, max_len, use_color);
 }
 
-static char *group_get_help(RzCmd *cmd, RzCmdDesc *cd, bool use_color) {
+static char *group_get_help(RzCmd *cmd, RzCmdDesc *cd, RzConfig *config, bool use_color) {
+
+	bool scr_utf8 = config ? (strcmp(rz_config_get(config, "scr.utf8"), "true") == 0) : false;
+	bool scr_curvy = config ? (strcmp(rz_config_get(config, "scr.utf8.curvy"), "true") == 0) && scr_utf8 : false;
```

Comment:
use `rz_config_get_b` to get a boolean, and expect `config` to be non-null

Reply:
@wargio `config` can be `NULL` (see below when NULL is passed to `rz_cmd_get_help`). So the way to do it here is to return ascii `|` if `config` is NULL IMHO.

Reply:
agreed.